tests/uptane_test.cc

```diff
@@ -58,12 +58,12 @@ TEST(Uptane, VerifyDataBad) {
 
   config.storage.path = temp_dir.Path();
   boost::shared_ptr<INvStorage> storage = boost::make_shared<FSStorage>(config.storage);
-  Uptane::TufRepository repo("director", http.tls_server + "/director", config, storage, http);
-  Json::Value data_json = repo.getJSON("root.json");
+  Uptane::TufRepository repo("director", http.tls_server + "/director", config, storage);
+  Json::Value data_json = http.get(http.tls_server + "/director/root.json").getJson();
   data_json.removeMember("signatures");
 
   try {
-    repo.verifyRole(Uptane::Role::Root(), now, data_json);
```

Comment:
This 'now' should really be used, it is there to make the tests continue to work in the future.  I remember now that we had a problem where the tests suddenly started failing--I think the root cause was that repo.verifyRole was ignoring the 'now' argument.

The problem is that is tests depend on the current system time, then it is hard to simulate things like metadata expiry.

Reply:
Understand, I will hardcode today date